apps/design-land/src/app/image/image.component.html

```diff
@@ -1 +1,28 @@
-<daff-image src="/assets/card/uber-logo.svg" alt="uber logo" width="300" height="119"></daff-image>
\ No newline at end of file
+<daff-article>
+  <h1 daffArticleTitle>Image</h1>
+  <p daffArticleLead>
+    The image component is used to customize an image you want to display in your project. To those familiar with 
```

Comment:
`Images in Daffodil utilize the native HTML <code>img</code> element to display responsive images on a page and prevent content jumping while images are loading.`